labs/memory_bound/data_packing/solution.h

```diff
@@ -15,7 +15,11 @@ struct S {
   bool b;
 
   bool operator<(const S &s) const { return this->i < s.i; }
+  long long get_l() const { return l; }
```

Comment:
I think we can get away without accessor methods.

Reply:
The idea of placing accessors in the struct was to isolate `validate.cpp` from changes in order to solve this challenge. This way, the only place the user needs to adapt code is in the `init.cpp` and `solution.*` files. Alternative approach could be to provide free functions as accessors but that wouldn't really make a big difference. What do you think?